rossum-invoice-intelligence/README.md

```diff
@@ -0,0 +1,63 @@
+![Rossum Custom Skill](./sample_invoice.png)
+
+# Setup instructions
+
+ 1. Sign up for a [Rossum Developer](https://account.box.com/signup/n/developer#nql6m) account and obtain a Rossum secret key.
+
+ 2. Configure Serverless! 
+
+ `npm install -g serverless` 
+
+ Then in order to give Serverless access to your AWS account, you'll need to create a new IAM User with admin access. Here's how to do that:
```

Comment:
@eshaverma I'm wondering if we should be so specific here, as a) this will apply to every skill and b) serverless supports more than AWS. We can just move this to the root README, or link to a serverless framework tutorial?